src/selectedtests/app/dependencies.py

```diff
@@ -0,0 +1,53 @@
+"""Parsers used in selected tests API."""
+from typing import List
+
+from evergreen import EvergreenApi, Project
+from fastapi import Depends, HTTPException
+from starlette.requests import Request
+
+from selectedtests.datasource.mongo_wrapper import MongoWrapper
+from selectedtests.evergreen_helper import get_evg_project
+
+
+def changed_files_parser(changed_files: str) -> List[str]:
+    """
+    Get the list of strings specified in CSV format under change_files query parameter.
+
+    :param changed_files: The CSV string.
+    :return: The parsed list of strings.
+    """
+    return changed_files.split(",")
+
+
+def get_db(request: Request) -> MongoWrapper:
+    """
+    Get the configured database for the application.
+
+    :param request: The request needing access to the database.
+    :return: The database.
+    """
+    return request.app.state.db
```

Comment:
Would be curious to know how @dbradf feels about using app state to store the `db` variable. I think I am ok with this.

Reply:
This is probably ok.